__docs__/wonder-blocks-pill/pill.stories.tsx

```diff
@@ -0,0 +1,340 @@
+    const accentLarge = canvas.getByTestId("accent-large-test-id");
+
+    // Clickable pills
+    const neutralSmallClickable = canvas.getByTestId(
+        "neutral-small-clickable-test-id",
+    );
+    const accentSmallClickable = canvas.getByTestId(
+        "accent-small-clickable-test-id",
+    );
+    const neutralLargeClickable = canvas.getByTestId(
+        "neutral-large-clickable-test-id",
+    );
+    const accentLargeClickable = canvas.getByTestId(
+        "accent-large-clickable-test-id",
+    );
+
+    await expect(neutralSmall).toHaveStyle({
+        backgroundColor: Color.offBlack8,
+        color: Color.offBlack,
+        fontSize: 12,
+    });
+
+    await expect(accentSmall).toHaveStyle({
+        backgroundColor: Color.blue,
+        color: Color.white,
+        fontSize: 12,
+    });
+
+    await expect(neutralLarge).toHaveStyle({
+        backgroundColor: Color.offBlack8,
+        color: Color.offBlack,
+        fontSize: 16,
+    });
+
+    await expect(accentLarge).toHaveStyle({
+        backgroundColor: Color.blue,
+        color: Color.white,
+        fontSize: 16,
+    });
+
+    // Clickable styles
+    await userEvent.tab();
+    let computedStyle = getComputedStyle(neutralSmallClickable, ":hover");
+    await expect(computedStyle.outline).toBe("rgb(24, 101, 242) solid 2px");
+
+    await userEvent.tab();
+    computedStyle = getComputedStyle(accentSmallClickable, ":hover");
+    await expect(computedStyle.outline).toBe("rgb(24, 101, 242) solid 2px");
+
+    await userEvent.tab();
+    computedStyle = getComputedStyle(neutralLargeClickable, ":hover");
+    await expect(computedStyle.outline).toBe("rgb(24, 101, 242) solid 2px");
+
+    await userEvent.tab();
+    computedStyle = getComputedStyle(accentLargeClickable, ":hover");
+    await expect(computedStyle.outline).toBe("rgb(24, 101, 242) solid 2px");
+};
+
+export const WithTypography: StoryComponentType = () => (
+    <Pill size="large">
+        <BodySerif>
+            This is a {<Link href="#">link</Link>} inside the text of a pill.
+        </BodySerif>
+    </Pill>
+);
+
+WithTypography.parameters = {
+    docs: {
+        description: {
+            story: `Pills can have Wonder Blocks Typography elements
+                as children. In this example, the Pill has a Wonder Blocks
+                Typography \`BodySerif\` component as its child.
+                Note that this also allows you to include a link inside
+                the text, as is shown here.`,
+        },
+    },
+};
+
+export const WithStyle: StoryComponentType = () => {
+    const customStyle = {
+        backgroundColor: Color.offBlack,
+        color: Color.white,
+        paddingLeft: Spacing.xxLarge_48,
+        paddingRight: Spacing.xxLarge_48,
+
+        ":hover": {
+            outlineColor: Color.offBlack,
+        },
+
+        ":active": {
+            outlineColor: Color.offBlack64,
+            backgroundColor: Color.offBlack64,
+        },
```

Comment:
**praise:** I love how `Pill` uses pseudo-classes (e.g. `:hover`) so folks can override pseudo-classes as well here instead of having to workaround hard-coded class names (e.g.`.hovered`). 🥇